### ISSUE TITLE
Re-set window title when returning to xplr from e.g. editing a file

### ISSUE THREAD

**Issue by jghauser** [sayanarijit/xplr#510]
First of all, thanks a lot for this super neat piece of software!

When starting `xplr`, the window title is set to "xplr", which is great. But when I then edit a file in for example neovim (with action to -> open in editor) and then quit neovim to return to xplr, the title is set to whatever neovim set it.

This breaks my usecase because I like to have xplr in a floating window in sway which i send/get from a scratchpad with a keybind. To identify the window, I can use either the `title` or the `app_id`. The app_id is set by the terminal and can't be used to identify the window, leaving me the title. And because the title doesn't switch back to "xplr" I can't use that either once I've opened a file from xplr.

It would be great if xplr coudl re-set the title once I return to it.

I'm not quite sure whether this is an issue with `xplr` or `wezterm` (my terminal). The behaviour seems to work fine in the `kitty` terminal, but then again title changes seem to work alright in `wezterm` with programs other than `xplr`.

**Comment by sayanarijit** [comment]
Hi, I wasn't able to reproduce. Could you pls provide xplr and wezterm versions?

**Comment by sayanarijit** [comment]
Oh, and also the window manager.

**Comment by jghauser** [comment]
Thanks for the quick response! I think I've figured out that the issue isn't really with xplr but rather wezterm.

In my init.lua i have the following:
```lua
vim.opt.title = true
vim.opt.titlestring = [[%t - nvim]]
```
which makes nvim set window titles. Wezterm doesn't unset these when returning from neovim (I realised it also happens if i open neovim from the terminal and then close it again).

I'm going to close the issue. Thanks again!
